manifests/klusterlet/klusterlet-work-serviceaccount.yaml

```diff
@@ -3,3 +3,5 @@ kind: ServiceAccount
 metadata:
   name: {{ .KlusterletName }}-work-sa
   namespace: {{ .KlusterletNamespace }}
+imagePullSecrets:
+- name: {{ .ImagePullSecret }}
```

Comment:
this can be hardcoded, right?  No need to have a configuration option and make us look in two places.